QRScanner/QRScannerView.swift

```diff
@@ -238,7 +240,9 @@ public class QRScannerView: UIView {
 
             strongSelf.qrCodeImageView.frame = path.bounds
             strongSelf.qrCodeImageView.center = center
-        }
+            }, completion: { [weak self] _ in
+                self?.delegate?.success(qrCode)
```

Comment:
I think it is better to call this after animation.
This is the same as iOS 13+ design.